src/site/content/en/blog/enabling-shared-array-buffer/index.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title: Enabling Shared Array Buffer
+authors:
+  - jakearchibald
+description: >
+  The way to enable Shared Array Buffer is changing, but the good news is it works cross-browser and cross-platform.
+origin_trial:
+  url: https://developers.chrome.com/origintrials/#/view_trial/-TODO-CHANGE-THIS
+date: 2020-12-01
+updated: 2020-12-01
+hero: hero.jpg
+alt: Iframes, images, and windows being accessed by someone 'evil'
+tags:
+  - blog
+  - javascript
+  - shared array buffer
+  - coop
+  - coep
+  - spectre
+  - meltdown
+  - security
+feedback:
+  - api
+---
+
+It's fair to say `SharedArrayBuffer` has had a bit of a rough landing on the web, but things are settling down. Here's what you need to know:
+
+## In brief
+
+- `SharedArrayBuffer` will arrive in Android Chrome 88, but only for cross-origin isolated pages (details below).
+- `SharedArrayBuffer` is currently available in desktop Chrome, but from Chrome 91 it will be limited to cross-origin isolated pages.
+
+You can make a page _cross-origin isolated_ by serving the page with these headers:
+
+```http
+Cross-Origin-Embedder-Policy: require-corp
+Cross-Origin-Opener-Policy: same-origin
+```
+
```

Comment:
Might be good to mention the report-only mode as well to check for errors before enabling those headers.